### DISCUSSION TITLE
mrs_uav_system for teams of other type of robots

### DISCUSSION THREAD

**Discussion by bconvens** [ctu-mrs/mrs_uav_system#165] [General]
Hi,

Are you planning to expand the capabilities of your mrs_uav_system towards (mixed) teams of mobile ground robots or (mobile) manipulator robots? 

If not, do you think the whole system's structure should be redesigned from scratch or is their only a robot type specialization required that allows to resuse several of the existing packages?

Best,
Bryan

**Comment by klaxalk** [comment]
Hey, we are slowly moving with our development version of the system, which could now potentially control differential drive UGB and USV. But it is still very "raw".
